cmd/krel/cmd/release_notes.go

```diff
@@ -113,12 +141,85 @@ func runReleaseNotes() (err error) {
 	logrus.Infof("Using start tag %v", start)
 	logrus.Infof("Using end tag %v", tag)
 
-	_, err = releaseNotesFrom(start)
+	if releaseNotesOpts.sigReleaseFork == "" && releaseNotesOpts.jsonFile == "" && releaseNotesOpts.mdFile == "" {
+		logrus.Fatal("No output selected. You have to specify at least one of --md-file --json-file or --sigrelease-fork")
```

Comment:
Surface the error w/ `errors.New()` instead of `logrus.Fatal()`

Reply:
Done, all first level errors are now created with errors.New, while I'm wrapping others which are happening further inside.